server/model/article.py

```diff
@@ -0,0 +1,19 @@
+from typing import Optional, Union
+from server.model.base import Model
+
+
+class Article(Model):
+    db_name = 'articles'
+
+    schema = {
+        'name': str,
+        'description': str,
+        'available_units': int,
+        'price': Union[int, float],
```

Comment:
podes usar def my_func(number: Real) ya que no pincha con por ejemplo decimal:
https://docs.python.org/3/library/numbers.html#numbers.Real

o para este caso simplemente usar float porque acepta ints:
https://www.python.org/dev/peps/pep-0484/#the-numeric-tower
"when an argument is annotated as having type float, an argument of type int is acceptable"